chain/chain/src/metrics.rs

```diff
@@ -42,6 +42,14 @@ pub static NUM_ORPHANS: Lazy<IntGauge> =
 pub static HEADER_HEAD_HEIGHT: Lazy<IntGauge> = Lazy::new(|| {
     try_create_int_gauge("near_header_head_height", "Height of the header head").unwrap()
 });
+pub static TAIL_HEIGHT: Lazy<IntGauge> =
+    Lazy::new(|| try_create_int_gauge("near_tail_height", "Height of tail").unwrap());
+pub static CHUNK_TAIL_HEIGHT: Lazy<IntGauge> =
+    Lazy::new(|| try_create_int_gauge("near_chunk_tail_height", "Height of chunk tail").unwrap());
+pub static FORK_TAIL_HEIGHT: Lazy<IntGauge> =
+    Lazy::new(|| try_create_int_gauge("near_fork_tail_height", "Height of fork tail").unwrap());
+pub static GC_STOP_HEIGHT: Lazy<IntGauge> =
+    Lazy::new(|| try_create_int_gauge("near_gc_stop_height", "Target height of gc").unwrap());
```

Comment:
Could you also add code which is executed when node starts and which updates those metrics. Without that, counters won’t be exported until gc happens. We already have that issue with HEADER_HEAD_HEIGHT, see https://github.com/near/nearcore/issues/6409.

Reply:
I set TAIL_HEIGHT, CHUNK_TAIL_HEIGHT, FORK_TAIL_HEIGHT but not GC_STOP_HEIGHT when node starts, because I think the last one is related to the gc process and should only be set when gc is initiated.